src/core/client/stream/tabs/Profile/Settings/DeleteAccount/DeleteAccountContainer.tsx

```diff
@@ -0,0 +1,138 @@
+
+import DeleteAccountModal from "./DeleteAccountModal";
+
+import styles from "./DeleteAccountContainer.css";
+
+interface Props {
+  viewer: DeleteAccountContainer_viewer;
+  settings: DeleteAccountContainer_settings;
+}
+
+const DeleteAccountContainer: FunctionComponent<Props> = ({
+  viewer,
+  settings,
+}) => {
+  const cancelAccountDeletion = useMutation(CancelAccountDeletionMutation);
+
+  const [deletePopoverVisible, setDeletePopoverVisible] = useState(false);
+
+  const showPopover = useCallback(() => {
+    setDeletePopoverVisible(true);
+  }, [setDeletePopoverVisible]);
+  const hidePopover = useCallback(() => {
+    setDeletePopoverVisible(false);
+  }, [setDeletePopoverVisible]);
+
+  const cancelDeletion = useCallback(() => {
+    cancelAccountDeletion();
+  }, [cancelAccountDeletion]);
+
+  const { locales } = useCoralContext();
+
+  const deletionDate = viewer.scheduledDeletionDate
+    ? Intl.DateTimeFormat(locales, {
+        year: "numeric",
+        month: "numeric",
+        day: "numeric",
+        hour: "numeric",
+        minute: "numeric",
+        second: "numeric",
+      }).format(new Date(viewer.scheduledDeletionDate))
+    : null;
+
+  return (
+    <div className={styles.root}>
+      <DeleteAccountModal
+        open={deletePopoverVisible}
+        onClose={hidePopover}
+        userID={viewer.id}
+        scheduledDeletionDate={viewer.scheduledDeletionDate}
+        organizationEmail={settings.organization.contactEmail}
+      />
+
+      <Localized id="profile-settings-deleteAccount-title">
+        <Typography variant="heading3" className={styles.title}>
+          Delete My Account
+        </Typography>
+      </Localized>
+      <Localized id="profile-settings-deleteAccount-description">
+        <Typography variant="bodyCopy" className={styles.section}>
+          Deleting your account will permanently erase your profile and remove
+          all your comments from this site.
+        </Typography>
+      </Localized>
+
+      {!deletionDate && (
+        <Button variant="outlined" size="small" onClick={showPopover}>
+          <Icon size="sm" className={styles.icon}>
+            cancel
+          </Icon>
+          <Localized id="profile-settings-deleteAccount-requestDelete">
+            <span>Request account deletion</span>
+          </Localized>
+        </Button>
+      )}
+      {deletionDate && (
+        <>
+          <Localized
+            id="profile-settings-deleteAccount-cancelDelete-description"
+            $date={deletionDate}
+          >
+            <Typography variant="bodyCopy" className={styles.section}>
+              {`You have already submitted a request to delete your account.
+              Your account will be deleted on ${deletionDate}.
+              You may cancel the request until that time.`}
+            </Typography>
+          </Localized>
+          <Button variant="filled" size="small" onClick={cancelDeletion}>
+            <Localized
```

Comment:
Localization wrapper should be around the entirety of the Button to provide the `title` attribute rather than multiple localization wrappers inside the button. Maybe @cvle has some idea to simplify this?

Reply:
Yeah, for this one, I'd sync with @cvle if you can to find a good solution!